sympy/core/sympify.py

```diff
@@ -51,6 +51,41 @@ class CantSympify(object):
     pass
 
 
+def convert_numpy_types(a):
```

Comment:
I would make this private (start the name with an underscore `_`).

Reply:
This needs `@doctest_depends_on` since the doctest depends on NumPy (search the codebase for examples on how to use it).

Reply:
I tried making ```convert_numpy_types()``` private, but that doesn't work because it gets imported and used by ```numbers.py``` per the change above. I'll leave it public for now (unless I'm missing something).

Reply:
Also, I'm having some trouble with ```@doctest_depends_on```. I searched the codebase and inferred that I need to add ```from sympy.utilities.decorator import doctest_depends_on``` to ```sympify.py``` and a ```@doctest_depends_on``` statement later on, but I get an error whenever I add the import to ```sympify.py```. Specifically:

```>>> import sympy
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "sympy/__init__.py", line 58, in <module>
    from .core import *
  File "sympy/core/__init__.py", line 4, in <module>
    from .sympify import sympify, SympifyError
  File "sympy/core/sympify.py", line 11, in <module>
    from sympy.utilities.decorator import doctest_depends_on
  File "sympy/utilities/__init__.py", line 4, in <module>
    from .iterables import (flatten, group, take, subsets,
  File "sympy/utilities/iterables.py", line 11, in <module>
    from sympy.core import Basic
ImportError: cannot import name Basic```

Any ideas?